akka-docs-dev/rst/scala/stream-flows-and-basics.rst

```diff
@@ -109,7 +109,7 @@ There are various ways to wire up different parts of a stream, the following exa
 
 Back-pressure explained
 -----------------------
-Akka Streams implements an asynchronous non-blocking back-pressure protocol standardised by the `Reactive Streams`_
+Akka Streams implement an asynchronous non-blocking back-pressure protocol standardised by the `Reactive Streams`_
```

Comment:
Then maybe all the other chapters are wrong, too :) We need to decide whether the name is "Akka Stream" (matches package and artifact names) or "Akka Streams" (which is how we use it in 99% of the cases).